.github/PULL_REQUEST_TEMPLATE/regular-member-pull-request.md

```diff
@@ -0,0 +1,17 @@
+---
+name: "\U0001F680 Request to become a Regular Member"
+about: You'd like to participate in the Cross Project Council's work as a Regular Member. Yay!
+---
+
+<!--
+We are thrilled that you'd like to participate in our work in a meaningful way as a Regular Member of the Cross Project Council.
+
+Requirements and rules around becoming a Regular Member can be found in our Governance:
+https://github.com/openjs-foundation/cross-project-council/blob/master/GOVERNANCE.md#approving-and-onboarding-regular-members
+
+Please fill out information below.
+
+Thanks!
+-->
+
+Please describe your recent participation in the work of the foundation or its member projects as described in the [CPC Governance requirements for becoming a Regular Member](https://github.com/openjs-foundation/cross-project-council/blob/master/GOVERNANCE.md#approving-and-onboarding-regular-members)
```

Comment:
I dont have a lot of experience with pull request templates. Is it good practice to have a generic pull request template similar to when you have issue templates?

Reply:
if we expect other kinds of PRs, then probably yeah

Reply:
should L17 be in a comment as well?

Reply:
I left L17 in the body because I thought it should be clear in the PR what association there is with the potential new member.

I've been looking further into PR templates and having this template in the PULL_REQUEST_TEMPLATE directory means it isn't the default one and can only be loaded via query params. I dont understand this fully and am inclined to remove it but perhaps we should leave it in and create an issue to work out pull request templates. I think it would be ideal to have a pull request template for this use case but I would need to put in some more work to get this functioning properly. It isn't clear to me how to make this work currently.